openpype/hosts/houdini/plugins/create/create_review.py

```diff
@@ -123,3 +129,46 @@ def get_pre_create_attr_defs(self):
                       minimum=0.0001,
                       decimals=3)
         ]
+
+    def set_colorcorrect_to_default_view_space(self,
+                                               instance_node):
+        """Set ociocolorspace to the default output space."""
+
+        if os.getenv("OCIO") is None:
+            # No OCIO, skip setting ociocolorspace
+            return
+
+        # if there's OCIO then set Color Correction parameter
+        # to OpenColorIO
+        instance_node.setParms({"colorcorrect": 2})
+
+        self.log.debug("Get default view colorspace name..")
+
+        default_view_space = self.get_default_view_space()
+        instance_node.setParms(
+            {"ociocolorspace": default_view_space}
+        )
+
+        self.log.debug(
+            "'OCIO Colorspace' parm on '{}' has been set to "
+            "the default view color space '{}'"
+            .format(instance_node, default_view_space)
+        )
+
+        return default_view_space
+
+    def get_default_view_space(self):
+        """Get default view space for ociocolorspace parm."""
+
+        from openpype.pipeline.colorspace import get_display_view_colorspace_name  # noqa
+        from openpype.hosts.houdini.api.lib import get_color_management_preferences  # noqa
+
+        data = get_color_management_preferences()
+        config_path = data.get("config")
+        display = data.get("display")
+        view = data.get("view")
+
+        default_view_space = get_display_view_colorspace_name(config_path,
+                                                              display, view)
```

Comment:
I wonder whether it makes more sense to use whatever is configured in settings as view instead of what Houdini is currently using a 'view' transform. From an artist perspective I'd expect to at least get an output matching whatever my Houdini view looks like. Maybe we should be focused more at the houdini view itself being configured the way we want and to have the review instance just use whatever the houdini view is doing? Or am I using some odd reversed logic?

Reply:
as far as I know, Houdini uses `scene_linear` role as render space
which is not correct when exporting reviews, as mentioned these [screenshots](https://github.com/ynput/OpenPype/pull/5322#issuecomment-1677259981)

Houdini needs three pieces of information to properly publish review in the correct output transform: 
1. View  (returned by houdini py api)
2. display (returned by houdini py api)
3. output transform (not supported, but it's displayed when clicking dropdown menu [screenshot](https://github.com/ynput/OpenPype/pull/5322#issuecomment-1672044600 ) )

That's why I added `get_display_view_colorspace_name` function  which returns the default output transform of the default view and default display

if everything kept in defaults, review output will be in `srgb`

This function call only populates review output transform with the default value 
```
 default_view_space = get_display_view_colorspace_name(config_path,
                                                              display, view)
```